src/modules/app-info/app-info.reducer.ts

```diff
@@ -0,0 +1,27 @@
+import { createFeatureSelector } from '@ngrx/store';
+
+import { AppInfoModel } from './app-info.model';
+import * as appInfoActions from './app-info.actions';
+
+export const initialState: AppInfoModel = {
+  versionNumber: 'z.z.z',
```

Comment:
Perhaps a better default for this, something like `VERSION_NOT_LOADED`?

Reply:
Good point